File: Modul 6/SOAL 5/PRAK505-2210817210002-GustiMuhammadNaufalRazin.c
#include <stdio.h>

void Biodata(int tahun_lahir,int A,int B){
	int tahun_sekarang = 2020;
    int usia = tahun_sekarang - tahun_lahir;
    printf("\nPerkenalkan Nama Saya : %s\n",A);
    printf("Umur Saya : %d\n",usia);
    printf("Saya Adalah Angkatan : %d\n",tahun_sekarang);
    printf("Asal Saya Dari : %s\n",B);
}
int main() {
    for(int i=0; i<=1; i++){
	int tahunLahir;
	    char A[20], B[15];
	    scanf(" %d",&tahunLahir);
	    scanf(" %[^\n]%*c",&A);
	    scanf(" %[^\n]%*c",&B);
	Biodata(tahunLahir, A, B);}
	return 0;
}

File: Modul 6/SOAL 5/PRAK505-2210817210002-GustiMuhammadNaufalRazin.py
def biodata(tahun_lahir, Namaku, Asal):
    tahun_sekarang = 2020
    usia = tahun_sekarang - tahun_lahir
    print("\nPerkenalan Nama Saya : ", Namaku)
    print("Umur Saya : ", usia)
    print("Saya adalah angkatan : ", tahun_sekarang)
    print("Asal saya dari : ", Asal)
    print("\n")

def main():
    for i in range(2):
     tahunLahir = int(input())
     A = str(input())
     B = str(input())
     biodata(tahunLahir, A, B)
main()